lstchain/mc/sensitivity.py

```diff
@@ -130,7 +129,7 @@ def process_mc(dl2_file, mc_type):
 def process_real(dl2_file):
 
     events = pd.read_hdf(dl2_file, key = dl2_params_lstcam_key)
-    obstime_real = get_obstime_real(events)
+    obstime_real = get_effective_time(events)[0]
```

Comment:
Please note that with the recent changes in get_effective_time, this will now be an astropy quantity with units of seconds. Make sure this does not make trouble later!

Reply:
Currently tests are failing because get_effective_time is getting an events table with no "dragon_time"

Reply:
`simulated_dl2` does not have `dragon_time`. Is that the reason?

Reply:
Seems so. If it is simulated, it shouldn't need either to calculate the effective time.

Reply:
I updated the test to include an artifical dragon_time in the test files, just as it was donde with delta_t. Now the test are passing but  not codecov/project and I can't understand why... I'm not yet very used to this codecov. Does anybody has an idea about what can I do to pass that test?

Reply:
tests are passing now, anything else missing for this PR?

Reply:
On my side it is finished.